cmd/cortex/main.go

```diff
@@ -155,3 +158,19 @@ func DumpYaml(cfg *cortex.Config) {
 		fmt.Printf("%s\n", out)
 	}
 }
+
+// expandEnv replaces ${var} or $var in config according to the values of the current environment variables.
+// The replacement is case-sensitive. References to undefined variables are replaced by the empty string.
+// A default value can be given by using the form ${var:default value}.
+func expandEnv(config []byte) []byte {
```

Comment:
The config can include passwords as well. What happens if a password, specified in the config, contains the `$` symbol? I think it will end up in a breaking change because after the `expandEnv()` the `$` will be removed even if the environment variable name is not set.

Maybe could be safer enabling the env var expansion via a CLI flag (disabled by default)?

Reply:
I added `config.expand-env` (defaults to false) options.